rskj-core/src/main/java/co/rsk/peg/NonStandardErpRedeemScriptBuilderFactory.java

```diff
@@ -0,0 +1,34 @@
+package co.rsk.peg;
+
+import co.rsk.bitcoinj.core.NetworkParameters;
+import org.ethereum.config.blockchain.upgrades.ActivationConfig;
+import org.ethereum.config.blockchain.upgrades.ConsensusRule;
+
+public class NonStandardErpRedeemScriptBuilderFactory {
+
+    private NonStandardErpRedeemScriptBuilderFactory() {
+    }
+
+    public static ErpRedeemScriptBuilder defineNonStandardErpRedeemScriptBuilder(
```

Comment:
```suggestion
    public static ErpRedeemScriptBuilder getNonStandardErpRedeemScriptBuilder(
```
That's a more common name, `get`